packages/api-kit/tests/e2e/getTransactionConfirmations.test.ts

```diff
@@ -21,13 +19,6 @@ describe('getTransactionConfirmations', () => {
       .to.be.rejectedWith('Invalid safeTxHash')
   })
 
-  it.skip('should return an empty array if the safeTxHash is not found', async () => {
```

Comment:
Why was this test removed?

Reply:
I didn't think much here and removed it because it was being skipped. It was likely skipped because it was throwing an error if you didn't set a correct hash, so I fixed it and changed the hash